README.md

```
@@ -137,6 +137,10 @@ For VCPKG integration, check out the example [here](https://github.com/luxonis/d
 Note that the VCPKG integration is using a custom branch of DepthAI and we plan to integrate the support for it into the main branch in the future and add it to the official VCPKG repository.
 
 
+### Telemetry
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)